f3ast/structure.py

```diff
@@ -193,7 +214,8 @@ def plot_slices(self, *args, **kwargs):
             axes: Matplotlib axes.
         """
         points = self.get_sliced_points()
-        ax = points3d(points, *args, **kwargs)
+        ax, sc = points3d(points, *args, **kwargs)
+        plt.colorbar(sc, shrink=0.8)
```

Comment:
Nice, but I feel like this should be added to `points3d` function, not here. Then you also don't need `matplotlib` import here